times.go

```diff
@@ -25,28 +25,50 @@ func Time(then time.Time) string {
 	return RelTime(then, time.Now(), "ago", "from now")
 }
 
-var magnitudes = []struct {
+type Seconds int64
```

Comment:
It's not clear to me why Seconds is better than time.Duration.  I may not
be paying enough attention, but creating a type here seems unnecessarily
limiting.

On Sun, Aug 23, 2015 at 1:01 AM Dmitri Shuralyov <notifications@github.com>
wrote:

> In times.go
> <https://github.com/dustin/go-humanize/pull/26#discussion_r37703458>:
>
> > @@ -25,28 +25,50 @@ func Time(then time.Time) string {
> >  	return RelTime(then, time.Now(), "ago", "from now")
> >  }
> >
> > -var magnitudes = []struct {
> > +type Seconds int64
>
> This is an exported type, it should be documented. Perhaps something like
> this:
>
> // Seconds represents the elapsed time between two instants as an int64 second count.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dustin/go-humanize/pull/26/files#r37703458>.
>